types/uuid.go

```diff
@@ -137,7 +137,7 @@ func (a UUIDArray) String() string {
 		}
 		switch {
 		case a.IsNull(i):
-			o.WriteString("(null)")
+			o.WriteString(array.NullValueStr)
 		default:
 			uuidStr, err := uuid.FromBytes(arr.Value(i))
 			if err != nil {
```

Comment:
Also seems like `a.Value(i)` could be used here now instead (just a refactor basically)

Reply:
I'd like to scope it to values mostly (maybe NullValueStr belongs to a separate PR?)

Reply:
Followed up in 8b840b7 for other types (json had to use valuestr, though)